tutorial/README.md

```diff
@@ -200,3 +200,14 @@ You can also open the latest job/activity log by:
 5. Click on Save variables button
 
 ![](images/image20-how-to-view-and-override-variables-from-editor.png)
+
+**How to perform cost estimation?**
+
+1. Download [tfcost Binary](https://github.com/IBM-Cloud/terraform-cost-estimator/releases) & put the binary in your PATH
+2. Select View > Command Palette... from VS Code menu bar ( Keyboard Shortcut: ⇧⌘P )
+3. Select IBM Cloud Schematics workspace: Estimate Cost
+4. Select target API to deploy. Here select cloud.ibm.com from the picker
+5. Enter API key. Type in your cloud.ibm.com API key and hit enter
+6. A new WebView with estimated cost of all resources is displayed
+
```

Comment:
These steps won't be sufficient for a non-terraform users who is using this tutorial first time for cost estimation. We need to provide a example repo in the steps which will help users to try this out. Specifically a step between 1 and 2 that says to clone a repo url and then open it in vscode and then run the cost estimation command

Reply:
Also the screenshot which is added below needs to be changed to a screenshot showing the file explorer with the example template from the previous comment.

Reply:
We have added the additional line in the tutorial to try out the command with a template after clonning
also we updated the screenshot accordingly